src/pages/home/Home.js

```diff
@@ -18,63 +17,79 @@ import config from 'config/constants';
-            <Button onClick={noop} className={styles.callToAction}>
-              {t('home-header-cta')}
-            </Button>
-          </header>
-          <Image className={styles.heroImage} alt={t('home-header-media-alt')} src={heroImage} />
-        </div>
-      </section>
+    return (
+      <Fragment>
+        <Navbar />
+        <section className={styles.hero}>
+          <div className={styles.heroInner}>
+            <header className={styles.heroTextWrapper}>
+              <h1 className={styles.heroTitle}>
+                <span>{t('home-header-title')}</span>
+              </h1>
+              <h2 className={styles.heroSubtitle}>{t('home-header-subtitle')}</h2>
+              <Button onClick={() => null} className={styles.callToAction}>
+                {t('home-header-cta')}
+              </Button>
+            </header>
+            <Image className={styles.heroImage} alt={t('home-header-media-alt')} src={heroImage} />
+          </div>
+        </section>
 
-      <main className={styles.content}>
-        <HomeSection
-          id="about"
-          title={t('home-about-title')}
-          text={t('home-about-text')}
-          media={<Image src={ImageAbout} alt={t('home-about-media-alt')} />}
-        />
-        <HomeSection
-          id="mission"
-          title={t('home-mission-title')}
-          text={t('home-mission-text')}
-          media={<Image src={ImageMission} alt={t('home-mission-media-alt')} />}
-        />
-        <HomeSection
-          id="contact"
-          title={t('home-contact-title')}
-          media={<img src={ImageContact} alt={t('home-contact-media-alt')} />}
-        >
-          <p>
-            <Interpolate i18nKey="home-contact-text" slackOrg={slackOrg} email={emailLink} />
-          </p>
-          <SocialMedia />
-        </HomeSection>
-      </main>
-      <Footer />
-    </Fragment>
-  );
+        <main className={styles.content}>
+          <HomeSection
+            id="about"
+            title={t('home-about-title')}
+            text={t('home-about-text')}
+            media={<Image src={ImageAbout} alt={t('home-about-media-alt')} />}
+          />
+          <HomeSection
+            id="mission"
+            title={t('home-mission-title')}
+            text={t('home-mission-text')}
+            media={<Image src={ImageMission} alt={t('home-mission-media-alt')} />}
+          />
+          <HomeSection
+            id="contact"
+            title={t('home-contact-title')}
+            media={<img src={ImageContact} alt={t('home-contact-media-alt')} />}
+          >
+            <p>
+              <Interpolate
+                i18nKey="home-contact-text"
+                slackOrg={
+                  <a
+                    href={config.contact.SLACK_URL}
+                    className={styles.slackOrg}
+                    onClick={(e) => {
+                      e.preventDefault();
+                      this.setState({ showModal: true });
```

Comment:
Do we need to show the modal here? I think this link should go directly to slack, right? *Maybe I'm missing something*

Reply:
I think it was the old implementation If not, I placed this not even knowing LOL. I'll update it! I do think too it should go to slack